docs/queries/filters.md

```diff
@@ -39,54 +53,75 @@ Natural language notation:
 - `last friday`
 - `14 days ago`
 - `in two weeks`
+- `14 October` (the current year will be used)
+- `May` (1st May in the current year will be used)
 
 Note that if it is Wednesday and you write `tuesday`, Tasks assumes you mean "yesterday", as that is the closest Tuesday.
 Use `next tuesday` instead if you mean "next tuesday".
 
-When the day changes, relative dates like `due today` are re-evaluated so that the list stays up-to-date.
-
-### Date ranges
-
-`<date range>` can be given in 2 ways.
+### Searching date ranges
 
 {: .released }
 Date range searches were introduced in Tasks 1.26.0.
 
-#### Date ranges in formal notation
+Tasks allows date searches to specify a pair of dates, `<date range>` .
 
-`<date range>` may be specified as 2 valid dates in `YYYY-MM-DD` format. Dates on either end are included, that is it is an inclusive search.
+These searches are inclusive: the dates at either end are found by the search.
 
-`before <date range>` will match before the earliest date of the range. `after <date range>` will match after the latest date of the range.
+### Absolute date ranges
+
+`<date range>` may be specified as 2 valid dates in `YYYY-MM-DD` format.
+
+Dates on either end are included, that is, it is an inclusive search.
+
+- `before <date range>` will match before the earliest date of the range.
```

Comment:
probably worth adding `exclusively` ie the `<date range>` will not be included. And same for the line below.

Reply:
I was thinking that `match before the earliest date` meant that it did not match `the earliest date`

Reply:
Ah yes this works)